master/static/srv/allocation_raw.sql

```diff
@@ -0,0 +1,79 @@
+WITH const AS (
+    SELECT
+        $1 :: timestamp AS day_start,
+        $2 :: timestamp AS day_end
+),
+-- Workloads that had any overlap with the target interval, with their start and end times bounded
+-- to the interval.
```

Comment:
Just front the comment: should just bound the end time to the interval? So we always count full workloads but never miss workloads that spanned interval boundaries?

Reply:
Can't quite tell what you mean here -- let's discuss it live. Though I did notice the comment is now a bit out of date; the time column does only count the time during the period, but the stand and and time are the originals. I think that makes sense, though I can see alternatives. Let's go over that as well.